### ISSUE TITLE
Relying more on ray?

### ISSUE THREAD

**Issue by klieret** [GageDeZoort/gnn_tracking#63]
If we want to get rid of tracking losses per epoch by ourselves with `ray`'s report function, we also have to use its training call which would probably look somewhat like so:

```python
scaling_config = ScalingConfig(num_workers=3)
# If using GPUs, use the below scaling config instead.
# scaling_config = ScalingConfig(num_workers=3, use_gpu=True)
trainer = TorchTrainer(
    train_loop_per_worker=tcn_trainer.train,
    scaling_config=scaling_config,
)
result = trainer.fit()
```

Advantage is that we might get something out of the scaling config? And that it then becomes apparently very, very easy to hook up tensorboard, mlflow and more by simply adding 

```python
run_config=RunConfig(
        callbacks=[
            MLflowLoggerCallback(experiment_name="train_experiment"),
            TBXLoggerCallback(),
        ],
    ),
```

to the above ([more info]( https://docs.ray.io/en/latest/train/dl_guide.html#example-logging-to-mlflow-and-tensorboard)).

**Comment by klieret** [comment]
Can be done in external steering file, no need to bake it in at this point
